entity/contents.cc

```diff
@@ -484,6 +489,26 @@ SolidStrokeContents::Cap SolidStrokeContents::GetStrokeCap() {
   return cap_;
 }
 
+static Scalar CreateBevel(
```

Comment:
Nit: Can you document that this returns direction? Or perhaps rename this to `CreateBevelAndGetDirection` perhaps?